feature-libs/product/configurators/common/styles/mixins/_configurator-required-error-msg.scss

```diff
@@ -0,0 +1,10 @@
+@mixin cx-required-error-msg {
+  .cx-required-error-msg {
+    color: var(--cx-color-danger);
+    font-size: 14px;
+
+    cx-icon {
+      padding-inline-end: 5px;
+    }
+  }
+}
```

Comment:
You are absolutely right.
In avoid code duplication I defined this mixin because there are 2 places where it will be used in the attribute header and footer. I can rename it :)

Reply:
Done!